src/components/Main/NewSurveyPage/NewSurveyPage.component.jsx

```diff
@@ -62,7 +65,28 @@ const NewSurveyPage = props => {
     }
   };
 
-  const handleQeustionSave = e => {
+  const handleSurveySave = e => {
+    e.preventDefault();
+    const questions = surveyState.questions;
+    const value = e.target.innerText;
+    if (questions.length === 0) {
+      alert('Добавьте и сохраните хотя бы один вопрос');
+    } else {
+      const survey = cloneDeep(surveyState);
+      survey.id = survey.name.split(' ').join('_');
```

Comment:
This does not ensure unique ids. You can use a library for this purpose, e.g. https://www.npmjs.com/package/shortid

Reply:
done, only used nanoid instead